### PR TITLE
New community projects section

### PR DESCRIPTION
**Pull request by guybedford**
This updates the README to include a new Community Projects section, showcasing projects that build on top of SystemJS.

For now I've just got jspm, single-spa and js-env.

@dmail @joeldenning please see the copy included and project any updates to the text as necessary. I will wait for your confirmation before merging.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,19 +100,22 @@ Say `main.js` depends on loading `'lodash'`, then we can define an import map:
 </script>
 ```
 
-### Browser transpilation
 
-To load ES modules directly in older browsers with SystemJS we can install and use the Babel plugin:
+## Community Projects
```
[the remark this answers is not in the document]
Comment (joeldenning) [reply README.md]: 👍 yeah looks good

**Comment by LarsDenBakker** [comment]
Would you consider adding https://github.com/open-wc/open-wc/tree/master/packages/es-dev-server? You can write standard es modules and import maps, the dev server converts them to es-module-shim or system-js based on browser support. Takes care of the bootstrapping code in the index.html too.

**Comment by guybedford** [comment]
@LarsDenBakker sure I've added that as well.

**Comment by dmail** [comment]
I appreciate the mention that's perfect thank you :)

**Pull request merged**